### DISCUSSION TITLE
Error when Saving Row

### DISCUSSION THREAD

**Discussion by mmalessa80** [Budibase/budibase#10343] [Help] [answered]
I've been getting an error in the console log when attempting to save a row, however I can't determine what the error actually is to attempt to fix it. Here is a screenshot if anyone could please guide me where to look to come up with additional details. Unless I ran out of some quota under the free plan, which on occasion I have received an error popup mentioning something about a quota.

![image](https://user-images.githubusercontent.com/106758851/232867623-82ed7570-4b34-4436-9853-ca42ffb7a8f8.png)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @mmalessa80,

Could you provide a screenshot of your network payload whenever you are trying to save, 500 suggests internal server error. I've supplied some example screenshots to help direct you on this.

<img width="836" alt="Screenshot 2023-04-19 at 10 50 32" src="https://user-images.githubusercontent.com/126772285/233038337-6e315c93-47d7-4edc-b275-624cff6dd4f4.png">

<img width="840" alt="Screenshot 2023-04-19 at 10 50 40" src="https://user-images.githubusercontent.com/126772285/233038363-18293551-89f4-468b-8a0c-34ab90cf4252.png">

<img width="844" alt="Screenshot 2023-04-19 at 10 50 52" src="https://user-images.githubusercontent.com/126772285/233038402-1e49c8d8-29b2-4bc1-a204-38d0dcf6bb1d.png">

Thanks!

**Reply by ConorWebb96** [reply]
Could you also screenshot the response within the network tab as well, apologises forgot to mention it.

**Reply by mmalessa80** [reply]
It appears this was related to a column setting for database vs value type in the field itself. However, the error did not correctly reflect this until I tried saving close to a dozen times. One of my columns is of type tinyInt (Number) in my MySQL DB to recognize a Boolean condition, so my form field I set up as Radio Select with values 0 and 1. However, when saving the row it was passing the values as "0" or "1". So in the form 'save row' I had to set the column value = Number(Field Value) and that apparently did the trick. As stated though, I tried close to a dozen times and the error didn't appear correctly until later that actually stated that there was a type mismatch.
